docs/en/observability/whats-new.asciidoc

```diff
@@ -122,15 +122,15 @@ Logs have been integrated into various contextual experiences in previous releas
 * {observability-guide-all}/7.13/whats-new.html[7.13]: APM service instances allow direct links to the logs per container, pod or host instance.
 * {observability-guide-all}/7.12/whats-new.html[7.12]: The ability to add logs viewer to any {kib} dashboard. With the release of ECS logging libraries
 in 7.11, APM introduced logs as part of a distributed end-to-end trace view.
```

Comment:
Could you please make ```With the release of ECS logging libraries
in 7.11, APM introduced logs as part of a distributed end-to-end trace view.``` a different bullet point and format accordingly?

Reply:
Thanks. Updated it.